tests/back/Integration/IntegrationTestsBundle/Loader/FixturesLoader.php

```
@@ -32,68 +33,28 @@
-
-    /** @var DoctrineJobRepository */
-    private $doctrineJobRepository;
-
-    /** @var FixtureJobLoader */
-    private $fixtureJobLoader;
-
-    /** @var AclManager */
-    private $aclManager;
-
-    /** @var ProductIndexerInterface */
-    private $productIndexer;
-
-    /** @var ProductModelIndexerInterface */
-    private $productModelIndexer;
-
-    /** @var ClientRegistry */
-    private $clientRegistry;
-
-    /** @var Client */
-    private $esClient;
-
-    /** @var Connection */
-    private $dbConnection;
-
-    /** @var string */
-    private $databaseHost;
-
-    /** @var string */
-    private $databaseName;
-
-    /** @var string */
-    private $databaseUser;
-
-    /** @var string */
-    private $databasePassword;
-
-    /** @var string */
-    private $sqlDumpDirectory;
-
-    /** @var \Elasticsearch\Client */
-    private $nativeElasticsearchClient;
-
-    /** @var MeasurementInstaller */
-    private $measurementInstaller;
+    private KernelInterface $kernel;
+    private DatabaseSchemaHandler $databaseSchemaHandler;
+    private SystemUserAuthenticator $systemUserAuthenticator;
+    private Application $cli;
+    private ReferenceDataLoader $referenceDataLoader;
+    private Filesystem $archivistFilesystem;
+    private DoctrineJobRepository $doctrineJobRepository;
+    private FixtureJobLoader $fixtureJobLoader;
+    private AclManager $aclManager;
+    private ProductIndexerInterface $productIndexer;
+    private ProductModelIndexerInterface $productModelIndexer;
+    private ClientRegistry $clientRegistry;
+    private Client $esClient;
+    private Connection $dbConnection;
+    private string $databaseHost;
+    private string $databaseName;
+    private string $databaseUser;
+    private string $databasePassword;
+    private string $sqlDumpDirectory;
+    private \Elasticsearch\Client $nativeElasticsearchClient;
+    private MeasurementInstaller $measurementInstaller;
+    private TransportInterface $transport;
```

Be careful about refactoring non-related code, you can raise unexpected errors.